crates/matrix-sdk-ui/CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ All notable changes to this project will be documented in this file.
   [`Timeline::send_reply()`] now takes an event ID rather than a `RepliedToInfo`.
   `Timeline::replied_to_info_from_event_id` has been made private in `matrix_sdk`.
   ([4842](https://github.com/matrix-org/matrix-rust-sdk/pull/4842))
+- [**breaking**] Allow sending media as (thread) replies. The reply behaviour
```

Comment:
Not sure it's a breaking change this time, is it? :D

Reply:
Err .. no, I think it's not. 🤦‍♂️